spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/servlet/ServletManagementChildContextConfiguration.java

```diff
@@ -145,14 +149,18 @@ private String getContextPath(ManagementServerProperties managementServerPropert
 
 	abstract static class AccessLogCustomizer implements Ordered {
 
-		private static final String MANAGEMENT_PREFIX = "management_";
+		protected final ManagementServerProperties managementServerProperties;
 
-		protected String customizePrefix(String prefix) {
+		AccessLogCustomizer(ManagementServerProperties managementServerProperties) {
+			this.managementServerProperties = managementServerProperties;
+		}
+
+		protected String customizePrefix(String prefix, String managementAccesslogPrefix) {
```

Comment:
You can drop this parameter and use `this.managementServerProperties.getAccesslog().getPrefix()` instead.

Reply:
Good catch, done